tests/python/rest_api/test_labels.py

```diff
@@ -663,6 +679,37 @@ def test_can_patch_label_field(self, source, admin_user, param, newvalue):
             user, label["id"], patch_data, expected_data=expected_data, ignore_fields=ignore_fields
         )
 
+    @parametrize("source", _TestLabelsPermissionsBase.source_types)
+    def test_can_patch_attribute_name(self, source, admin_user):
+        user = admin_user
+        label = next(
+            iter(
+                self._labels_by_source(
+                    self.labels, source_key=self._get_source_info(source).label_source_key
+                ).values()
+            )
+        )[0]
+
+        if label.get("attributes"):
```

Comment:
Only a label with attributes should be selected here. Overwise the test will always pass if a label has no attributes and there will be no point in such a test.

Reply:
Done